demo/admin/src/products/ProductForm.tsx

```diff
@@ -104,6 +106,7 @@ export function ProductForm({ id }: FormProps) {
                       id: filteredData.manufacturerCountry?.addressAsEmbeddable.country,
                   }
                 : undefined,
+            lastCheckedAt: filteredData.lastCheckedAt ? parseISO(filteredData.lastCheckedAt) : null,
```

Comment:
It's odd that we need date-fns here but not in the submit handler. Why not use `new Date(filteredData.lastCheckedAt)` instead?

Reply:
The [submit handler](https://github.com/vivid-planet/comet/pull/2639/files#diff-c02e0d95b73fa156370bc5e9e28ba0af249cf2601561de2fb384d7d31576ab0fR139) uses it too.
I chose this, because the dev tool errors suggested `parseISO()` as the solution, when nothing is adapted in `initialValues` or in `handleSubmit`:  

<img width="697" alt="Screenshot 2024-10-22 at 10 30 40" src="https://github.com/user-attachments/assets/15a45178-d2f4-4f83-8083-2fdd845db385">

Is `new Date()` the better solution here?

Reply:
I meant that you're not using format or similar from date-fns in the submit handler. So using new Date() seems more logical to me. That's also what the old generator does. @nsams what do you think?

Reply:
@nsams what do you think?

Reply:
@nsams @johnnyomair Have you been talking about this topic at some time?

Reply:
Please try using `new Date(filteredData.lastCheckedAt)` instead of parseISO and see if it still works.

Reply:
It works!